.github/CONTRIBUTING.md

```diff
@@ -2,20 +2,32 @@
 
 If you've written a new formatter, added a new locale, or fixed a bug, your contribution is welcome!
 
-Before proposing a pull request, check the following:
+Before proposing a pull request, please read these contribution guidelines.
 
-* Your code should follow our coding standard. Run `make cs` to fix coding standard issues.
-* Unit tests should pass after your patch. Run the tests in your local environment (with `make test`) or check the continuous integration status for your pull request.
-* As much as possible, add unit tests for your code
-* Never use `rand()` in your provider. Faker uses the Mersenne Twister Randomizer, so use `mt_rand()` or any of the base generators (`randomNumber`, `randomElement`, etc.) instead.
-* If you add a new provider and it embeds a lot of data (e.g. first names in a new language), please add a `@see` PHPDoc to the reference you used for this list (example [in the `en_US` locale](https://github.com/FakerPHP/Faker/blob/main/src/Faker/Provider/en_US/Text.php#L35)). This will ease future updates of the list and debates about the most relevant data for this provider.
+## Issues & Pull Requests
+
+* Communicate in English on issues and pull requests.
+* Pull requests should only contain related changes.
+* Your code should follow [our coding standard](/.php_cs.dist). Run `make cs` to fix code style issues.
+* Your code should be covered by unit tests.
+* All checks must pass before your code is merged. Run `make test` to run unit tests.
+
+## Best Practices & Conventions
+
+When writing code for Faker, you should know about these conventions / best practices:
+
+* Never use `rand()` in your provider. Faker utilizes the Mersenne Twister Randomizer, so use `mt_rand()` or any of the base generators (`randomNumber`, `randomElement`, etc.) instead.
+* If you add a new provider which it embeds a lot of data (e.g. first names in a new language), please add a `@see` PHPDoc to the reference you used for this list (example [in the `en_US` locale](https://github.com/FakerPHP/Faker/blob/main/src/Faker/Provider/en_US/Text.php#L35)). This will ease future updates of the list and debates about the most relevant data for this provider.
```

Comment:
If you add a new provider which ~~it~~ embeds

Reply:
Fixed.